### DISCUSSION TITLE
#prompt got changed? Confused....

### DISCUSSION THREAD

**Discussion by amelsen** [scandum/tintin#121] [General]
Heya... 

wanted to start mudding again.. havent for 6-7 years.. and noticed my old prompt doesn't work? not sure whats wrong.

My old prompt setup looked like this:

#PROMPT {ppp} {--- TANK= $tank --- FOE= $foe --- WIMPYDIR= $wimpydir ---} {2} {}

Meaning, once done... I could write 'ppp' and the prompt would update? Now it doesn't seem to work anymore.. and the help text confuses me?

Can someone elaborate?

**Comment by scandum** [comment]
You'll need to change `{2}` to either `{}` or `{-2}`. The -2 means second line from the bottom, and 2 would be second line from the top.
